commit 66a66aa74ee262f03432f84de55ae314c391d9f5
Author: Sarangdhar Joshi <spjoshi@codeaurora.org>
Date:   Thu Feb 2 16:04:37 2017 -0800

    remoteproc: Drop qcom_scm_pas_supported() from adsp_probe()
    
    SCM call to check whether Peripheral Authentication Service (PAS) is
    supported returns false for ADSP on MSM8996. Drop this call from the
    driver so that the probe() function succeeds for ADSP PIL device.
    
    Signed-off-by: Sarangdhar Joshi <spjoshi@codeaurora.org>
    Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>

diff --git a/drivers/remoteproc/qcom_adsp_pil.c b/drivers/remoteproc/qcom_adsp_pil.c
index f1b147c96d44..49fe2f807e1d 100644
--- a/drivers/remoteproc/qcom_adsp_pil.c
+++ b/drivers/remoteproc/qcom_adsp_pil.c
@@ -337,11 +337,6 @@ static int adsp_probe(struct platform_device *pdev)
 	if (!qcom_scm_is_available())
 		return -EPROBE_DEFER;
 
-	if (!qcom_scm_pas_supported(desc->pas_id)) {
-		dev_err(&pdev->dev, "PAS is not available for subsystem\n");
-		return -ENXIO;
-	}
-
 	rproc = rproc_alloc(&pdev->dev, pdev->name, &adsp_ops,
 			    desc->firmware_name, sizeof(*adsp));
 	if (!rproc) {